addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyMissingScanRuleUnitTest.java

```diff
@@ -262,21 +296,45 @@ void shouldReturnExpectedMappings() {
                 is(equalTo(CommonAlertTag.OWASP_2017_A06_SEC_MISCONFIG.getValue())));
     }
 
+    @Test
+    void shouldReturnExampleAlerts() {
+        // Given / When
+        List<Alert> alerts = rule.getExampleAlerts();
+        long countMediums =
+                rule.getExampleAlerts().stream()
+                        .filter(alert -> Alert.RISK_MEDIUM == alert.getRisk())
+                        .count();
+        long countInfos =
+                rule.getExampleAlerts().stream()
+                        .filter(alert -> Alert.RISK_INFO == alert.getRisk())
+                        .count();
+        // Then
+        assertThat(alerts.size(), is(equalTo(3)));
+        assertThat(countMediums, is(equalTo(1L)));
+        assertThat(countInfos, is(equalTo(2L)));
+    }
+
     private void assertContentSecurityPolicyAlertRaised() {
         assertThat(alertsRaised.size(), is(1));
-        assertCSPAlertAttributes(alertsRaised.get(0), "", Alert.RISK_MEDIUM);
+        assertCSPAlertAttributes(alertsRaised.get(0), "", Alert.RISK_MEDIUM, "10038-1");
+    }
+
+    private void assertObsoleteSecurityPolicyAlertRaised() {
+        assertThat(alertsRaised.size(), is(1));
+        assertCSPAlertAttributes(alertsRaised.get(0), "obs.", Alert.RISK_INFO, "10038-2");
     }
 
-    private static void assertCSPAlertAttributes(Alert alert, String key, int expectedRisk) {
+    private static void assertCSPAlertAttributes(
+            Alert alert, String key, int expectedRisk, String alertRef) {
         assertThat(alert.getRisk(), is(expectedRisk));
         assertThat(alert.getName(), is(getLocalisedString(key + "name")));
         assertThat(alert.getDescription(), is(getLocalisedString(key + "desc")));
-        assertThat(alert.getReference(), is(getLocalisedString(key + "refs")));
```

Comment:
Was this removed on purpose? Reference/Refs are links for the issue. alertRefs are labels/identifiers for the alert.

Reply:
I've reinstated it, but had to move it because two alerts used the non-keyed version.